spec/deaf_grandma_spec.rb

```diff
@@ -10,9 +10,13 @@
   
   it "says 'NOT SINCE 1964!' when we yell" do
     #implement your test here
+    expect(script.speak("SPEAK WHAT")).to eq "NOT SINCE 1964!"
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
     #implement your test here
+    script.speak("BYE")
+    script.speak("BYE")
```

Comment:
^^^this but just because you'd get lampooned if it were implemented this way in prod...anywhere :)